cybershell/vulnerability_kb.py

```
@@ -0,0 +1,740 @@
+    def load_custom_payloads(self):
+        """Load custom payloads from files"""
+        custom_payloads_dir = self.kb_path / "custom_payloads"
+        custom_payloads_dir.mkdir(exist_ok=True)
+        
+        # Load JSON payloads
+        for json_file in custom_payloads_dir.glob("*.json"):
+            try:
+                with open(json_file, 'r') as f:
+                    data = json.load(f)
+                    self._process_custom_payload_data(data)
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)